src/Response/ParcelShopsResponse.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\OverseasBundle\Response;
+
+use Answear\OverseasBundle\Enum\StatusResult;
+use Answear\OverseasBundle\Response\DTO\Error;
+use Answear\OverseasBundle\Response\DTO\ParcelShop;
+use Answear\OverseasBundle\Response\DTO\Validation;
+use Symfony\Component\Serializer\Normalizer\DenormalizableInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class ParcelShopsResponse implements ResponseInterface, DenormalizableInterface
+{
+    /**
+     * @var ParcelShop[]
+     */
+    private array $data = [];
+    private StatusResult $status;
+    private ?Error $error = null;
+    /**
+     * @var Validation[]
+     */
+    private array $validations = [];
+
+    /**
+     * @return ParcelShop[]|null
+     */
+    public function getData(): ?array
```

Comment:
$data can't be null

Reply:
right, @see https://github.com/answear/overseas-bundle/pull/6/files#diff-bc8869e626ebc4d71c2de85673beb336d0bd0fedb97a76abc674f51d1b2a0470

there is no getter